lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -71,6 +73,7 @@
                         ['topics/:topic_id/search(/)', _.bind(this.searchTeams, this)],
                         ['topics/:topic_id/create-team(/)', _.bind(this.newTeam, this)],
                         ['topics/:topic_id/:team_id/edit-team(/)', _.bind(this.editTeam, this)],
+                        ['topics/:topic_id/:team_id/edit-team/manage-members(/)', _.bind(this.editTeamMembers, this)],
```

Comment:
Actually, the server errors are well handled (generic error message displayed in the top message area) for these 2 views.